openmetadata-ui/src/main/resources/ui/src/pages/tags/index.tsx

```diff
@@ -215,7 +215,7 @@ const TagsPage = () => {
   const createCategory = (data: CreateTagCategory) => {
     const errData = onNewCategoryChange(data, true);
     if (!Object.values(errData).length) {
-      createTagCategory(data)
+      createTagCategory({ ...data, name: trimStart(trimEnd(data.name)) })
```

Comment:
@Ashish8689 , Can we use the `trim` method here?

Reply:
@Sachin-chaurasiya  updated!